api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -110,72 +110,79 @@ public void setDescription(String description) {
 	}
 	
 	/**
-	 * @see org.openmrs.Auditable#getCreator()
+	 * @see org.openmrs.OpenmrsMetadata#getCreator()
 	 */
 	@Override
 	public User getCreator() {
 		return creator;
 	}
 	
 	/**
-	 * @see org.openmrs.Auditable#setCreator(org.openmrs.User)
+	 * @see org.openmrs.OpenmrsMetadata#setCreator(org.openmrs.User)
 	 */
 	@Override
 	public void setCreator(User creator) {
 		this.creator = creator;
 	}
 	
 	/**
-	 * @see org.openmrs.Auditable#getDateCreated()
+	 * @see org.openmrs.OpenmrsMetadata#getDateCreated()
 	 */
 	@Override
 	public Date getDateCreated() {
 		return dateCreated;
 	}
 	
 	/**
-	 * @see org.openmrs.Auditable#setDateCreated(java.util.Date)
+	 * @see org.openmrs.OpenmrsMetadata#setDateCreated(java.util.Date)
 	 */
 	@Override
 	public void setDateCreated(Date dateCreated) {
 		this.dateCreated = dateCreated;
 	}
 	
 	/**
-	 * @see org.openmrs.Auditable#getChangedBy()
+	 * @see org.openmrs.OpenmrsMetadata#getChangedBy()
+	 * @deprecated as of version 2.2
```

Comment:
Our convention has been to include what they should alternatively use. https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
The methods are actually going away with no replacement, I think I just have to add more details as to why it was necessary to remove the method.

Reply:
And to mention that we introduced a new interface that one would need to implement to in case they want to have a mutable class

Reply:
Actually the method javadocs have the `@see` the superclass where information as to why it's deprecated is included